### DISCUSSION TITLE
taro支持编译专有钉钉小程序吗

### DISCUSSION THREAD

**Discussion by wangshuo1779** [NervJS/taro#15096] [Q & A] [answered]
### 这个特性解决了什么问题？
使用taro框架开发，可以支持编译专有钉钉小程序

### 这个 API 长什么样？
支持编译专有钉钉小程序

**Comment by ZakaryCode** [comment, marked as answer]
https://github.com/NervJS/taro-plugin-platform-alipay-dd

**Comment by wangshuo1779** [comment]
专有钉钉和钉钉是两个东西，有差异的
